src/Accounts/Authentication/Config/ConfigInitializer.cs

```diff
@@ -190,6 +190,14 @@ private void RegisterConfigs(IConfigManager configManager)
                 true,
                 AzurePSDataCollectionProfile.EnvironmentVariableName,
                 new[] { AppliesTo.Az }));
+#if DEBUG || TESTCOVERAGE
+            configManager.RegisterConfig(new SimpleTypedConfig<bool>(
+                ConfigKeys.EnableTestCoverage,
+                "N/A",
```

Comment:
Let's make the help message meaningful just in case.
```suggestion
                "When enabled, the test framework will calculate and output a test coverage report.",
```

Reply:
Help message added.